docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,457 @@
+
+    OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_START_INDEX =
+        OE_SGX_ENDORSEMENT_FIELD_CRL_END_INDEX + 1,
+
+    OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_END_INDEX =
+        OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_START_INDEX +
+        OE_SGX_ENDORSEMENTS_CRL_COUNT - 1,
+
+    OE_SGX_ENDORSEMENT_FIELD_QE_ID_INFO =
+        OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_END_INDEX + 1,
+
+    OE_SGX_ENDORSEMENT_FIELD_QE_ID_ISSUER_CHAIN,
+    OE_SGX_ENDORSEMENT_FIELD_CREATION_DATETIME
+
+} oe_sgx_endorsements_fields;
+```
+
+### <B>Private</B> SGX endorsement definitions
+
+`common/sgx/evidence.h`
+```C
+/*! \struct oe_sgx_endorsements
+ *
+ * \brief SGX endorsements structure
+ *
+ * The generic oe_endorsements_t structure is parsed and converted into this
+ * internal structure.  The order of the generic data elements should
+ * coincide with the order of the fields in this structure.
+ *
+ * Data format: All data comes from the Data Center Attestation Primitives(DCAP)
+ * Client.
+ *
+ * For Azure DCAP Client
+ * (https://github.com/microsoft/Azure-DCAP-Client/blob/master/src/dcap_provider.h)
+ * see **sgx_ql_revocation_info_t** and sgx_qe_identity_info_t.
+ *
+ * For Intel DCAP Client
+ * (https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/README.md)
+ * see TBD.
+ *
+ */
+typedef struct _oe_sgx_endorsements_t
+{
+    uint8_t* tcb_info;                      ///< TCB info, null-terminated JSON string
+    uint32_t tcb_info_size;                 ///< TCB Info size
+    uint8_t* tcb_issuer_chain;              ///< PEM format, null-terminated string
+    uint32_t tcb_issuer_chain_size;         ///< Size of the tcb_issuer_chain
+
+    ///< CRLs in DER format, null-terminated
+    ///<     crl[0] = CRL for the SGX PCK Certificate
+    ///<     crl[1] = CRL for the SGX PCK Processor CA
+    uint8_t* crl[OE_SGX_ENDORSEMENTS_CRL_COUNT];
+
+    ///< CRLs sizes
+    uint32_t crl_size[OE_SGX_ENDORSEMENTS_CRL_COUNT];
+
+    ///< PEM format, null-terminated string
+    uint8_t* crl_issuer_chain[OE_SGX_ENDORSEMENTS_CRL_COUNT];
+
+    ///< Size of each crl_issuer_chain
+    uint32_t crl_issuer_chain_size[OE_SGX_ENDORSEMENTS_CRL_COUNT];
+
+    uint8_t* qe_id_info;                    ///< QE Identity info, null-terminated JSON string
+    uint32_t qe_id_info_size;               ///< QE Identity size
+    uint8_t* qe_id_issuer_chain;            ///< PEM format, null-terminated string
+    uint32_t qe_id_issuer_chain_size;       ///< Size of qe_id_issuer_chain
+
+    uint8_t* creation_datetime;             ///< Time the endorsements were generated,
+                                            ///< null-terminated string
+    uint32_t create_datetime_size;          ///< The size of creation_datetime.
+
+} oe_sgx_endorsements_t;
+```
+
+### New <B>Public</B> Attestation functions
+
+These functions supersede the existing functions:
+1. `oe_get_evidence()` supersedes `oe_verify_report()`
```

Comment:
>verify [](start = 38, length = 6)

oe_get_report()

Reply:
The `RemoteAttestationCollaterals.md` design doc is already merged in, but Ill fix that typo in a future PR.